docs/interfaces/postgres.rst

```diff
@@ -267,8 +267,8 @@ implemented, for example::
     cr> COMMIT
     COMMIT OK, 0 rows affected  (... sec)
 
-Since CrateDB does not support transactions, both the ``COMMIT`` and ``BEGIN``
-statement and any of its parameters are ignored.
+Since CrateDB does not support transactions, the ``COMMIT`` and ``BEGIN/START``
+statements and any of its parameters are ignored.
```

Comment:
```suggestion
CrateDB will silently ignore the ``COMMIT TRANSACTION``, ``BEGIN TRANSACTION``,
and ``START TRANSACTION statements and all respective parameters.
```

reworded this to use the [active voice](https://www.grammarly.com/blog/active-vs-passive-voice/). I also expanded the statement literals for precision and clarity

I removed the "since" part because I think its redundant given the rest of this subsection